src/deepsparse/v2/pipeline.py

```diff
@@ -51,12 +52,14 @@ def __init__(
         router: Router,
         schedulers: List[OperatorScheduler],
         pipeline_state: PipelineState = None,
+        # [TODO] middlewares: List[]
```

Comment:
let's take out any changes to pipeline.py for this PR and add it in the next PR that actually introduces middleware. do not want to leave the v2 branch in incomplete state

Reply:
ok